apps/files/src/components/FilesAppBar.vue

```diff
@@ -5,9 +5,9 @@
       <div class="uk-width-expand">
         <div class="uk-flex">
           <oc-breadcrumb id="files-breadcrumb" :items="breadcrumbs" v-if="showBreadcrumb" home></oc-breadcrumb>
-          <span class="uk-margin-small-left" v-if="showBreadcrumb && currentFolder.privateLink">
+          <span class="uk-margin-small-left" v-if="showBreadcrumb && currentFolder && currentFolder.privateLink">
           <oc-icon name="ready" v-show="linkCopied" />
-          <oc-icon id="files-permalink-copy" name="link" v-clipboard="() => currentFolder.privateLink"
+          <oc-icon id="files-permalink-copy" name="link" v-clipboard="() => currentFolder && currentFolder.privateLink"
```

Comment:
Is the check for current folder here necessary when we check for it already in the parent?

Reply:
yes, this solves the JS error. I guess the vue was still rendering or evaluating this piece while the current folder was not loaded yet

Reply:
ah, got your point. I didn't check

Reply:
fixed